posthog/api/async_migration.py

```diff
@@ -78,6 +83,19 @@ def trigger(self, request, **kwargs):
         trigger_migration(migration_instance)
         return response.Response({"success": True}, status=200)
 
+    @action(methods=["POST"], detail=True)
+    def resume(self, request, **kwargs):
+        migration_instance = self.get_object()
+        if migration_instance.status != MigrationStatus.Errored:
```

Comment:
this seems weird. maybe we need some new status? I'd feel that something that errored can't be resumed.